cmd/cmd_root_execute.go

```diff
@@ -47,6 +50,22 @@ func init() {
 	rootCmd.AddCommand(cmdVersion)
 }
 
+// ExecuteCommand executes commands, intended for testing
```

Comment:
would be nice if we don't have to expose a function just for testing purposes but I'm not sure there is any other hook in (I took this from cobra's unit test code)